src/util/messageBot.ts

```diff
@@ -0,0 +1,43 @@
+import FluorineClient from '@classes/Client';
+import { Message } from 'discord.js';
+
+export async function messageBot(client: FluorineClient, message: Message) {
+    let bot = 0;
+    const authorDate = new Date(message.author.createdTimestamp + 12096e5);
+    const memberDate = new Date(message.member.joinedTimestamp + 1800000);
+    const url = message.content.match(/\bhttps?:\/\/\S+/giu);
+    const urlBoolean = false;
```

Comment:
why is this a thing? it's only used on an if check which isn't ever triggered because it's always false